doc/release/upcoming_changes/28741.change.rst

```diff
@@ -0,0 +1,18 @@
+Re-enabled the capability to override the following functions:
+ * :func:`numpy.strings.center`
+ * :func:`numpy.strings.capitalize`
+ * :func:`numpy.strings.decode`
+ * :func:`numpy.strings.encode`
+ * :func:`numpy.strings.expandtabs`
+ * :func:`numpy.strings.lust`
```

Comment:
What is `numpy.strings.lust`? This is causing the documentation build to fail. Personally, I would not try to list all the changed functions.

Reply:
I think it is meant to be `ljust`. @byrdie - I sort-of agree with @mattip that there is no need to list all functions. Maybe more important to say is that all functions in `np.strings` can again be overridden. (If not, it is a bug!) But also OK to just correct the typo here!

Reply:
@mattip, good catch! I've updated the docs like you suggested.